x-pack/legacy/plugins/siem/public/components/open_timeline/index.test.tsx

```diff
@@ -53,23 +42,26 @@ describe('StatefulOpenTimeline', () => {
       </ThemeProvider>
     );
 
-    await wait();
-    wrapper.update();
+    const componentProps = wrapper
+      .find('[data-test-subj="open-timeline"]')
+      .last()
+      .props();
 
-    expect(getStateChildComponent(wrapper).state).toEqual({
+    expect(componentProps).toEqual({
+      ...componentProps,
       itemIdToExpandedNotesRowMap: {},
       onlyFavorites: false,
       pageIndex: 0,
       pageSize: 10,
-      search: '',
+      query: '',
```

Comment:
I am confused why it change from search to query, did we change the model of the store too

Reply:
i will answer your question in the `open_timeline/index.tsx` below